code/code.py

```diff
@@ -279,27 +283,37 @@ def code_state_while():
     def code_state_return():
         """Inserts return statement"""
 
+    def code_break():
+        """Inserts break statement"""
+
+    def code_next():
+        """Inserts next statement"""
+
+    def code_true():
+        """Insert True value"""
+
+    def code_false():
```

Comment:
I didn't see these implemented anywhere?

Reply:
My bad, I've now implemented them for R and Pyhon (https://github.com/knausj85/knausj_talon/pull/220/commits/485d775296ede11cc93fca056d325fb3fa54be91).